pandas/core/frame.py

```diff
@@ -7140,17 +7142,16 @@ def corr(self, method="pearson", min_periods=1):
         Parameters
         ----------
         method : {'pearson', 'kendall', 'spearman'} or callable
-            Method of correlation:
-
+            Method of correlation.
```

Comment:
You need to keep the blank line between the first sentence and the bullet points (I *think* that's the reason sphinx is complaining).

BTW, I think the colon (`:`) also makes sense to keep before a bullet point list.

Reply:
yup, thanks. That was just me playing with it to see if I could get the validation to accept it